pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -37,7 +37,7 @@
 , blas
 , enableContrib ? true
 
-, enableCuda ? config.cudaSupport && stdenv.hostPlatform.isx86_64
```

Comment:
Oh I dropped this without thinking. Why do we ignore `config` on non-x86_64 in the first place?

Reply:
https://github.com/NixOS/nixpkgs/commit/c4adeddb5f8e945517068968d06ea838b7c24bd3

> config.cudaSupport can be true and cudatoolkit doesn't work with i686,
> which can happen inside pkgsi686Linux wrapping.

OK, so this wasn't about aarch64 😅

Reply:
Also, why would anyone set `config.cudaSupport = true` on an i686 device?

Reply:
I should move this to a separate commit though